spec/action/help_spec.rb

```diff
@@ -32,7 +30,9 @@
         -h, --help        Print this help message.
         -V, --version     Print the version number.
         -f, --files       Print the list of configuration files used.
-        --just-view FILE  Display FILE, given with full path (no searching).
+        --just-view       Display NAME which is a full path (no searching).
+        --print-completion=SHELL
```

Comment:
I want `--print-completion` to be coherent with `--just-view` regarding the requirement of `=`.